services/email/index.js

```diff
@@ -11,6 +11,8 @@ var path = require('path');
 var customerHtmlTemplates = {
   error: fs.readFileSync(
     path.resolve(__dirname, './templates/customer/html/error.mus')).toString('utf8'),
+  'error-triage': fs.readFileSync(
```

Comment:
Could clean this up a bit by sticking it in it's own var and avoiding the duplicate code.

But it's not really important.